### PR TITLE
Update android-intune.md

### PR DESCRIPTION
**Pull request by msbemba**
Added steps to add storage permissions tasks.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9024

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/android-intune.md [hunk 1]
```diff
@@ -151,6 +151,14 @@ Follow the steps below to add Microsoft Defender for Endpoint app into your mana
...
 
         :::image type="content" source="images/android-create-app.png" alt-text=" The Associated app details pane" lightbox="images/android-create-app.png":::
+     
+    1. Select **Permissions > Add**. From the list, select the available app permissions > **OK**.
+    2. Select an option for each permission to grant with this policy:
+
+
+       - **Prompt** -  Prompt the user to accept or deny.
+       - **Auto grant** - Automatically approve without notifying the user.
+       - **Auto deny** - Automatically deny without notifying the user.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/android-intune.md]:
```suggestion
       - **Prompt** -  Prompts the user to accept or deny.
       - **Auto grant** - Automatically approves without notifying the user.
       - **Auto deny** - Automatically denies without notifying the user. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Pull request merged**